src/app/query/bitcoin/ordinals/use-inscription.query.ts

```diff
@@ -11,8 +11,11 @@ async function getInscriptionMetadata(path: string) {
   const validatedData = await ordApiXyzGetInscriptionByInscriptionSchema.validate(data);
   return validatedData;
 }
+
+export const queryPrefix = 'inscription-metadata';
```

Comment:
Would it be better to call these `queryKey`, not critical to change but just curious what `Prefix` means?

Reply:
Also, could these be an enum in one place?

Reply:
We can def use a better name. I chose `queryPrefix` b/c `queryKey` is the entire array, and this is just the first part or topmost part of that key.

Reply:
> Also, could these be an enum in one place?

My Github UI was lagging and just saw this. Good idea, added them to enum. Open to feedback on location & naming.